java/src/jmri/jmrit/decoderdefn/DecoderFile.java

```diff
@@ -144,10 +145,44 @@ public DecoderFile(String mfg, String mfgID, String model, String lowVersionID,
         } else {
             _manufacturerID = "-1";
         }
-
         _productID = productID;
     }
 
+    /**
+     * Create a mechanism to manipulate a decoder definition.
+     *
+     * @param mfg manufacturer name
+     * @param mfgID manufacturer's NMRA manufacturer number, typically a "CV8" value
+     * @param model decoder model designation
+     * @param lowVersionID decoder version low byte, where applicable
+     * @param highVersionID decoder version high byte, where applicable
+     * @param family decoder family name, where applicable
+     * @param filename filename of decoder XML definition
+     * @param developerID SV2 developerID number (8 bits)
+     * @param manufacturerID SV2 manufacturerID number (8 bits)
+     * @param productID (typically) SV2 product ID number (16 bits)
```

Comment:
Is this method for LocoNet SV2 protocol only? The JavaDoc gives that impression.

Reply:
I expect it is, but reduced the SV2 reference in latest commit, thanks

Reply:
I recommend that you add LocoNet. For example:
`@param productID (typically) LocoNet SV2 product ID number (16 bits)`

The reason is that it's very hard for anyone without knowledge in LocoNet to understand what SV2 stands for. If it says `LocoNet SV2`, then you know what to search for.